config/am.py

```diff
@@ -53,6 +53,68 @@
 NUMERALS = union(byte.DIGIT,
                  utils.GEEZ_NUMERALS)
 
+REDUCE_H = union(cross("ሐ", "ሀ"),
```

Comment:
Can we use "string_map" here?

Reply:
Switched to string_map across the board! It did turn out that I was just missing some of the various parentheses